services/ui-src/src/components/MultiSelect/index.tsx

```diff
@@ -0,0 +1,124 @@
+import * as CUI from "@chakra-ui/react";
+import React, { useState } from "react";
+import { BsSearch } from "react-icons/bs";
+import { useController, useFormContext } from "react-hook-form";
+
+export interface ICheckbox {
+  value: string;
+  label: string;
+  isVisible: boolean;
+}
+interface Props {
+  multiSelectList: ICheckbox[];
+  name: string;
+}
+
+export const MultiSelect = ({ multiSelectList, name }: Props) => {
+  const { control } = useFormContext();
+  const { field } = useController({
+    name,
+    control,
+  });
+
+  let sourceList = [...multiSelectList];
+  sourceList.sort((a, b) => (a.label > b.label ? 1 : -1));
+  const [multiSelects, setMultiSelects] = useState<ICheckbox[]>(sourceList);
```

Comment:
Let's turn this into a one line statement
```suggestion
  const [multiSelects, setMultiSelects] = useState<ICheckbox[]>(multiSelectList.sort((a, b) => (a.label > b.label ? 1 : -1)));
```

Reply:
good suggestion. Done